Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN ./build.sh install_rpms
 COPY ./ /root/containerbuild/
 RUN ./build.sh write_archive_info
 RUN ./build.sh install_anaconda
+RUN git submodule update --init
```

Comment:
I think we may want this before `write_archive_info` conceptually but it may not matter today.

Reply:
It would actually be nice if there was a way to just detect and fail at the beginning of the build. We could just 

```
# Make sure git submodule for mantle has been properly initialized
COPY mantle/README.md /dev/null
```
and that would fail very early on and not waste any developer time

Reply:
>  It would actually be nice if there was a way to just detect and fail at the beginning of the build.

:+1: that'd be my vote

If you put this before `write_archive_info` it will break in the Openshift case. 

My feeling on this is that most use-cases for COSA involve pulling the container. Developer-local builds of the full-container are slow. And a common work-flow for devs use bind mounts over-top of an extant image; rebuilding is too painful for most.